dtfs-central-api/src/v1/controllers/portal/gef-facility/update-facility.controller.js

```diff
@@ -30,3 +30,17 @@ exports.updateFacility = async (id, updateBody) => {
     return e;
   }
 };
+exports.updateFacility = updateFacility;
+
+exports.updateFacilityPut = async (req, res) => {
+  const facilityId = req.params.id;
+  const facilityUpdate = req.body;
+
+  const updatedFacility = await updateFacility(facilityId, facilityUpdate);
+
+  if (updatedFacility) {
+    return res.status(200).json(updatedFacility);
+  }
+
+  return res.status(401).send('The facility ID doesn\'t exist');
```

Comment:
Sorry, I've mentioned in the previous comment that this should be 404. 401 means they are not authorized.
This should be:
`return res.status(404).send({ status: 404, message: 'The facility ID doesn\'t exist' });`